README.md

```diff
@@ -525,6 +529,20 @@ For more information on how runtime works you can read its [README](https://gith
 By default it is not possible to define multiple configuration files within `RUNTIME_SUBDIRECTORY` referencing the same domain.
 To enable this behavior set `MERGE_DOMAIN_CONFIG` to `true`.
 
+### From http provider
+
+The Ratelimit service allows to load configuration from a http server. The endpoint can be configured via the settings package with the following environment variables:
+
+```
+HTTP_PROVIDER_ENABLED default:"false"
+HTTP_PROVIDER_ENDPOINT
+HTTP_PROVIDER_SUBPATH default:""
+HTTP_PROVIDER_POLL_INTERVAL default:"10"
+HTTP_PROVIDER_POLL_TIMEOUT default:"10"
+```
+
+Configuration will be reloaded intervally according to the value of `HTTP_POLL_INTERVAL` (second). `HTTP_PROVIDER_SUBPATH` is an array which allows you to have multiple endpoints for different config files.
```

Comment:
This needs a lot more documentation. Can you fully specify the proposed protocol and then we can go from there? Thanks.